src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -330,7 +330,9 @@ def energyConversion(dict_values, group):
                 "Asset %s has multiple input/output busses with a list of efficiencies. Reading list",
                 dict_values[group][asset][LABEL],
             )
-    return
+        else:
+            logging.debug(f"Not loading {group} asset {asset} from file")
+            compute_timeseries_properties(dict_values[group][asset])
```

Comment:
I am actually not sure what that debug message tells me.

Reply:
>     * You also created the function `C0.compute_timeseries_properties()`, which is not in the changelog. Have you written a test for that? [f0cad40](https://github.com/rl-institut/multi-vector-simulator/commit/f0cad40d86fa47d45e66895820b8d771a12021db)

I move this code into a function because I wanted to be able to execute this also when not loading timeseries from csv, I did not write tests for it (here the function applies standard `sum` and `max` to a timeseries. There was already a kind of local test of the normalized timeseries which I kept. testing that `sum` does the sum is already done by the builtin test suite on the `sum` function in that case.